runtime/kusama/src/lib.rs

```diff
@@ -940,6 +963,9 @@ construct_runtime! {
 		// System scheduler.
 		Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
 
+		// Runtime impl; this is mainly to inject an appropriate variant into the Event enum
+		Inclusion: inclusion::{Event<T>},
+
```

Comment:
It seems to me that all relevant paras modules should go in here?

Reply:
It's hard for me to judge what the effect of including, or excluding, the various paras modules would actually be.

Reply:
Added them in 06e960d.